layers/stateless/sl_ray_tracing.cpp

```diff
@@ -1062,6 +1062,94 @@ bool Device::ValidateAccelerationStructureBuildGeometryInfoKHR(const Context &co
                              "(%" PRIu64 ") must be less than or equal to 2^32-1.", geom.geometry.aabbs.stride);
             }
         }
+        if (geom.geometryType == VK_GEOMETRY_TYPE_SPHERES_NV) {
+            auto sphere_struct = reinterpret_cast<VkAccelerationStructureGeometrySpheresDataNV const *>(geom.pNext);
+            if (!enabled_features.spheres) {
+                skip |= LogError("VUID-VkAccelerationStructureGeometrySpheresDataNV-None-10429", device,
+                                 geometry_loc.pNext(Struct::VkAccelerationStructureGeometrySpheresDataNV),
+                                 "The spheres feature must be enabled");
+            }
+            if (sphere_struct->vertexStride > vvl::kU32Max) {
+                skip |= LogError("VUID-VkAccelerationStructureGeometrySpheresDataNV-vertexStride-10432", handle,
+                                 geometry_loc.pNext(Struct::VkAccelerationStructureGeometrySpheresDataNV).dot(Field::vertexStride),
+                                 "(%" PRIu64 ") must be less than or equal to 2^32-1.", sphere_struct->vertexStride);
+            }
+            if (sphere_struct->radiusStride > vvl::kU32Max) {
+                skip |= LogError("VUID-VkAccelerationStructureGeometrySpheresDataNV-vertexStride-10432", handle,
+                                 geometry_loc.pNext(Struct::VkAccelerationStructureGeometrySpheresDataNV).dot(Field::radiusStride),
+                                 "(%" PRIu64 ") must be less than or equal to 2^32-1.", sphere_struct->radiusStride);
+            }
+            if (sphere_struct->indexType != VK_INDEX_TYPE_UINT16 && sphere_struct->indexType != VK_INDEX_TYPE_UINT32 &&
+                sphere_struct->indexType != VK_INDEX_TYPE_NONE_KHR) {
+                skip |= LogError("VUID-VkAccelerationStructureGeometrySpheresDataNV-indexData-10437", handle,
+                                 geometry_loc.pNext(Struct::VkAccelerationStructureGeometrySpheresDataNV).dot(Field::indexType),
+                                 "is %s.", string_VkIndexType(sphere_struct->indexType));
+            }
+            if (sphere_struct->vertexData.hostAddress == nullptr && sphere_struct->vertexData.deviceAddress == 0) {
+                skip |= LogError("VUID-VkAccelerationStructureGeometrySpheresDataNV-vertexData-parameter", handle,
+                                 geometry_loc.pNext(Struct::VkAccelerationStructureGeometrySpheresDataNV).dot(Field::vertexData),
+                                 "must be a valid VkDeviceOrHostAddressConstKHR union");
```

Comment:
if I got this error message I would think "wait, what is valid?!?!?"

just put

```suggestion
                                 "must not be 0 or NULL");
```

for these